src/Subclasses/User.php

```diff
@@ -22,6 +22,14 @@ class User extends ParseUser implements Authenticatable
 {
     use CastsParseProperties;
 
+	/**
+     * @return string
+     */
+    public function getAuthIdentifierName()
+    {
+        return $this->getUsername();
```

Comment:
The intention of the method is not to return the identifier, it's to return the column name which stores the unique identifier. In a standard Laravel MySQL DB, this would be 'id'. In parse, the "column" (key since its mongo) is called objectId. So you shouldn't be returning the result of a getter method, you need to return the actual name of the column/key. In this case you should return the string 'objectId'